resources/sdk/purecloudjavascript/templates/ApiClient.mustache

```diff
@@ -275,16 +276,21 @@ class ApiClient {
 				reject(new Error('The client credentials grant is not supported in a browser.'));
 				return;
 			}
-
 			const headers = {
 				'Authorization': `Basic ${authHeader}`
 			}
+            let httpsAgent;
+            if (this.proxy !== undefined){
+                httpsAgent = new HttpsProxyAgent({
+                    proxy : `${this.proxy.protocol}://${this.proxy.host}:${this.proxy.port}`
+                });
+            }
```

Comment:
I also can't seem to get the formatting right, it looks right on any editor I open it on except here